src/app/app.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="container">
+  <form method="post" class="form" [formGroup]="form" (ngSubmit)="onSubmit()">
+    <!-- name -->
```

Comment:
cool, but not needed
let's try to keep code as clean as possible so don't need to use comments :)